### DISCUSSION TITLE
Can not connect  "sh: 1: ssh: not found"

### DISCUSSION THREAD

**Discussion by kvnsya** [spatie/ssh#91] [Q&A]
$process = Ssh::create('HOST', 'USER')->execute('ls -a');
dd($process->getErrorOutput()); 

This is my dump
"sh: 1: ssh: not found"

If i just use  $connection = ssh2_connect('HOST', 22); i dont have any problem to connect.

What could be the problem?

**Comment by deminy** [comment]
Command "ssh" is not available. You need to install an SSH client. e.g.,

```bash
apt-get install openssh-client --no-install-recommends
```
